src/components/container/RestingHeartRateCalendar/RestingHeartRateCalendar.tsx

```diff
@@ -10,7 +10,7 @@ import { queryDailyData, DailyDataType } from '../../../helpers/query-daily-data
 
 type HeartRateMap = { [key: string]: number };
 
-export type RestingHeartRateCalendarPreviewState = "WithData" | "NoData" | "Loading";
+export type RestingHeartRateCalendarPreviewState = "WithData" | "NoData" | "Loading" | "Combined" | "AppleHealth" | "Fitbit" | "Garmin";
```

Comment:
This definitely doesn't seem right; the "previewState" (or showPreviewData in this case) params are just for providing a preview of non-live data so I wouldn't conflate restricting it to Garmin / Fitbit / AppleHealth with this

I would add another prop for "namespace" that is "AppleHealth" | "Fitbit" | "Garmin" | "Combined" that allows you to control the namespace; the prop should default to "Combined"